cli/command/cluster/remove.go

```diff
@@ -0,0 +1,82 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2021-10-15
+ * Author: Jingli Chen (Wine93)
+ */
+
+package cluster
+
+import (
+	"fmt"
+
+	"github.com/opencurve/curveadm/cli/cli"
+	"github.com/opencurve/curveadm/internal/log"
+	"github.com/opencurve/curveadm/internal/tui/common"
+	cliutil "github.com/opencurve/curveadm/internal/utils/cli"
+	"github.com/spf13/cobra"
+)
+
+type removeOptions struct {
+	clusterName string
+}
+
+func NewRemoveCommand(curveadm *cli.CurveAdm) *cobra.Command {
```

Comment:
the same as the clean cmd. how to deal with misoperation?

Reply:
> the same as the clean cmd. how to deal with misoperation?

i will improve it.